optuna_dashboard/ts/components/GraphTimeline.tsx

```diff
@@ -92,12 +92,7 @@ const plotTimeline = (trials: Trial[], mode: string) => {
     template: mode === "dark" ? plotlyDarkTemplate : {},
   }
 
-  const traces: Partial<plotly.PlotData>[] = []
-  for (const s of Object.keys(cm) as TrialState[]) {
-    const bars = trials.filter((t) => t.state === s)
-    if (bars.length === 0) {
-      continue
-    }
+  const makeTrace = (bars: Trial[], name: string, color: string) => {
```

Comment:
It is a very minor comment, but what about replacing `name` with `state`?

```suggestion
  const makeTrace = (bars: Trial[], state: string, color: string) => {
```